test/tests/tsdx-build.test.js

```diff
@@ -21,15 +20,11 @@ describe('tsdx build', () => {
     const output = shell.exec('node ../dist/index.js build --format esm,cjs');
 
     expect(shell.test('-f', 'dist/index.js')).toBeTruthy();
-    expect(
-      shell.test('-f', 'dist/build-default.cjs.development.js')
-    ).toBeTruthy();
-    expect(
-      shell.test('-f', 'dist/build-default.cjs.production.min.js')
-    ).toBeTruthy();
-    expect(shell.test('-f', 'dist/build-default.esm.js')).toBeTruthy();
-
-    expect(shell.test('-f', 'dist/index.d.ts')).toBeTruthy();
+    expect(shell.test('-f', 'dist/cjs/index.js')).toBeTruthy();
+    expect(shell.test('-f', 'dist/cjs/index.min.js')).toBeTruthy();
```

Comment:
I am confused. Is .min the production version?

Reply:
Sure. Why otherwise one will want minifying? and especially, in cjs/node target.